dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -122,22 +122,25 @@ public Host findDefaultHost(User user, boolean respectFrontendRoles) throws DotS
     @CloseDBIfOpened
     public Host resolveHostName(String serverName, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException {
         Host host = hostCache.getHostByAlias(serverName);
-        User systemUser = APILocator.systemUser();
-
-        if(host == null){
-
+        if (UtilMethods.isSet(host::getIdentifier)) {
```

Comment:
I think you need to do `UtilMethods.isSet(()->host.getIdentifier())` as this will throw an NPE if host comes back null.

Reply:
I know sonarqube does not like it but it is what it is.